interfaces/Portalicious/src/locale/messages.xlf

```diff
@@ -1579,6 +1579,27 @@
       <trans-unit id="general" datatype="html">
         <source>General</source>
       </trans-unit>
+      <trans-unit id="1233711102649541270" datatype="html">
+        <source>Ignore duplication</source>
+      </trans-unit>
+      <trans-unit id="3570735198692940811" datatype="html">
+        <source>Add a reason</source>
+      </trans-unit>
+      <trans-unit id="ignore-duplication" datatype="html">
+        <source>Ignore duplication</source>
+      </trans-unit>
+      <trans-unit id="ignore-duplicates-confirmation-message-first-2_duplicates" datatype="html">
+        <source>You&apos;re about to mark both registrations as unique.</source>
+      </trans-unit>
+      <trans-unit id="ignore-duplicates-confirmation-message-first-more_duplicates" datatype="html">
+        <source>You&apos;re about to mark <x equiv-text="{{ duplicateCount }}" id="INTERPOLATION"/> registrations as unique.</source>
```

Comment:
This pattern of useing the `@let`-syntax for these kind of variables is actually kind of nice.. As it prevents any translation from breaking the 'calculation'/expression that gets inserted...

And it makes it way easier to read/debug.